app/jobs/valkyrie_ingest_job.rb

```diff
@@ -36,6 +34,8 @@ def ingest(file:)
   def add_file_to_file_set(file_set:, file_metadata:)
     file_set.file_ids << file_metadata.id
     Hyrax.persister.save(resource: file_set)
+    # this triggers the re-index
+    Hyrax.publisher.publish('object.metadata.updated', object: file_set)
```

Comment:
i think we want to introduce `object.membership.updated`.

i'm working on this in this line: https://github.com/samvera/hyrax/pull/5375

Reply:
yeah that's why i marked this as draft, i wasn't sure if you wanted to tackle that first